File: bob.c
#include <stdio.h>
int main(){
	int test, n,m, step, extra;
	int total;
	int i=0;
	printf ("Enter the number of the testcases : ");
	scanf ("%d",&test);
	for (i=0;i<test;i++){
		printf ("Enter the value of N : ");
		scanf ("%d",&n);
		total = 4*n;
		m= n+1;
		step = total/m;
		extra = total%m;
		if (extra==0){
			printf ("Total no. of moves will be %d\n", step);
		}
		else if (extra>0){
			printf ("Total no. of moves will be %d\n", step+1);
		}
	}
	return 0;
}

File: bob.py
test = int(input("Enter the number of test cases : "))
for i in range (0,test):
    n = int(input("Enter the value of N : "))
    total = 4*n
    m = n+1
    steps = total//m
    extra = total%m
    if (extra == 0):
        print("The number of moves needed : {}".format(steps))
    elif (extra>0):
        print ("The number of moves needed : {}".format(steps+1))
